pages/set/_.vue

```diff
@@ -115,7 +115,7 @@
         </b-col>
       </b-row>
       <b-row
-        v-if="recommendations.length > 0"
+        v-if="enableRecommendations && recommendations.length > 0"
```

Comment:
Is it necessary to check whether recommendations are enabled or not here? I think it should suffice to only check for that when about to retrieve them, and here to just look for whether there are any set.

Reply:
Yes, you are right. I was thinking about removing it here this morning but kept it in for no particular reason. I have removed the check here.